src/sections/Community/Handbook/github-process.js

```diff
@@ -0,0 +1,463 @@
+                  <td>Meshery CLI related</td>
+                  <td>#3a0486</td>
+                </tr>
+                <tr>
+                  <td>component/ui</td>
+                  <td></td>
+                  <td>#3a0486</td>
+                </tr>
+                <tr>
+                  <td>component/filters</td>
+                  <td></td>
+                  <td></td>
+                </tr>
+                <tr>
+                  <td>component/patterns</td>
+                  <td></td>
+                  <td>#3a0486</td>
+                </tr>
+                <tr>
+                  <td>component/meshsync</td>
+                  <td>MeshSync related</td>
+                  <td>#3a0487</td>
+                </tr>
+                <tr>
+                  <td>component/meshery-perf</td>
+                  <td>Meshery-perf related</td>
+                  <td>#3a0488</td>
+                </tr>
+                <tr>
+                  <td>good first issue</td>
+                  <td>Good for newcomers</td>
+                  <td>#7057ff</td>
+                </tr>
+                <tr>
+                  <td>help wanted</td>
+                  <td>Extra attention is needed</td>
+                  <td>#35f48e</td>
+                </tr>
+                <tr>
+                  <td>issue/invite</td>
+                  <td>Send an invitation to the issue author to join the GH org</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/blocked</td>
+                  <td>Issue is blocked</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/design required</td>
+                  <td>Design specification needed</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/duplicate</td>
+                  <td>This issue or pull request already exists</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/invalid</td>
+                  <td>This doesn't seem right</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/stale</td>
+                  <td>Issue has not had any activity for an extended period of time</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/tweet</td>
+                  <td>Posts</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/remind</td>
+                  <td>Reminds the contributor towards the issue to be attentive</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
+                  <td>issue/willfix</td>
+                  <td>This issue will be worked on</td>
+                  <td>#eeeeee</td>
+                </tr>
+                <tr>
```

Comment:
Is there a forthcoming `issue/valid`, too?

Reply:
This might be of value if we chose to lock an issue during triage, or for a somewhat similar process, otherwise, if it's issue/invalid it's "closed". if it's issue/valid it's assigned (the use of the label in question might be slightly redundant otherwise?)